blocks/product-list-page/product-list-page.js

```diff
@@ -1,73 +1,91 @@
+/* eslint-disable import/no-unresolved */
+import { ProductListingPage } from '@dropins/storefront-search/dist/containers/ProductListingPage.js';
+import { render as provider } from '@dropins/storefront-search/dist/render.js';
```

Comment:
We will need to resolve why you are having to import from `dist` and not `@dropins/storefront-search/containers` for example.

Is that something you can work on in the scope of this PR?

Reply:
It is just easier for me to copy the folder over and add it to the path.

Reply:
Something like this should work: 
`cp -r ~/my-code-path/storefront-search/dist/* adobe-demo-store/scripts/__dropins__/storefront-search`